client/components/product-card/price-group.jsx

```diff
@@ -23,7 +23,9 @@ const ProductCardPriceGroup = props => {
 			{ isDiscounted && (
 				<PlanPrice currencyCode={ currencyCode } rawPrice={ discountedPrice } discounted />
 			) }
-			<span className="product-card__billing-timeframe">{ billingTimeFrame }</span>
+			<div className="product-card__billing-timeframe">
+				<div className="product-card__billing-timeframe-label">{ billingTimeFrame }</div>
```

Comment:
What is the reason for having more nesting here?

Reply:
Good catch! I needed it at one point but now it's redundant. I'm going to remove it.